entity-web/src/main/java/eu/europeana/entity/web/controller/BaseRest.java

```diff
@@ -228,6 +227,27 @@ protected String extractSolrFieldName(String query) {
 	return fieldName;
     }
 
+	/**
+	 * Validate language parameter
+	 * @param language
+	 * @throws ParamValidationException
+	 */
+	protected void validateLanguage(String language) throws ParamValidationException {
+	if (StringUtils.isEmpty(language))	{
+		return;
+	}
+	if (StringUtils.contains(language, WebEntityConstants.COMMA)) {
+		throw new ParamValidationException(I18nConstants.UNSUPPORTED_MULTIPLE_LANG_VALUE,
+				CommonApiConstants.QUERY_PARAM_LANG, language);
+	}
+
+	if (!language.matches(WebEntityConstants.LANG_REGEX)) {
```

Comment:
if we use the 2 letter codes only, we could use the Locale.getISOLanguages() to validate the request parameter

Reply:
done